src/contracts/interfaces/IPaymentCoordinator.sol

```diff
@@ -39,27 +40,12 @@ interface IPaymentCoordinator {
         IERC20[] tokens;
         // cumulative all-time earnings in each token
         uint256[] amounts;
+        //index in the merkle tree
+        uint256 index;
     }
 
-    // TODO: better define this event?
-    event PaymentReceived(address indexed receivedFrom, Payment payment);
-
-    // @notice Emitted when a new Merkle root is posted
-    event NewMerkleRootPosted(bytes32 root, uint256 height, uint256 confirmedAtBlockNumber, uint256 calculatedUpToBlockNumber);
-
-    /// @notice Array of roots of posted Merkle trees, as well as associated data like tree height
-    // MerkleRootPost[] public merkleRootPosts;
-    function merkleRootPosts(uint256 index) external view returns (MerkleRootPost memory);
-
     /// @notice Getter function for the length of the `merkleRootPosts` array
-    function merkleRootPostsLength() external view returns (uint256);
-
-    /// @notice Mapping token => recipient => cumulative amount *claimed*
-    // mapping(IERC20 => mapping(address => uint256)) public cumulativeTokenAmountClaimedByRecipient;
-    function cumulativeTokenAmountClaimedByRecipient(IERC20 token, address recipient) external view returns (uint256);
-
-    // @notice Constant that defines the share EigenLayer takes of all payments, in basis points
-    function EIGENLAYER_SHARE_BIPS() external view returns (uint256);
+    function merkleRootsLength() external view returns (uint256);
```

Comment:
was there a reason for renaming this? if we are calling the array `MerkleRootPosts`, I think `merkleRootPostsLength` is more technically accurate

Reply:
I've called the array merkleRoots

Reply:
doesn't each entry in the array contain more than just a root though? feel like this is the same semantic debate as what we call a 'Leaf'

Reply:
yeah in retrospect I agree with you on this as well.